requirements.txt

```diff
@@ -3,7 +3,7 @@ PyYAML
 requests>=2.0
 requests-mock>=0.7,<0.8
 django-crispy-forms>=1.6,<1.7
-django-selectable>=0.9.0,<1.0
+git+https://github.com/mlavin/django-selectable.git@master#egg=django-selectable
```

Comment:
Okay, so this one is very tempting, but it's a serious security threat because we're not pinning the version here.

How much work could it take to keep the current version and add a workaround for this `django-selectable` issue?